app/models/manageiq/providers/openstack/infra_manager/host/operations.rb

```diff
@@ -0,0 +1,24 @@
+module ManageIQ::Providers::Openstack::InfraManager::Host::Operations
+  def self.included(base)
```

Comment:
please instead of this, do include ActiveSupport::Concern

then we don't need the module InstanceMethods, just define module ClassMethods, active support will then handle that